src/lib/routes/admin-api/addon.ts

```diff
@@ -34,26 +33,18 @@ class AddonController extends Controller {
     }
 
     async getAddons(req: Request, res: Response): Promise<void> {
-        try {
-            const addons = await this.addonService.getAddons();
-            const providers = this.addonService.getProviderDefinitions();
-            res.json({ addons, providers });
-        } catch (error) {
-            handleErrors(res, this.logger, error);
-        }
+        const addons = await this.addonService.getAddons();
+        const providers = this.addonService.getProviderDefinitions();
+        res.json({ addons, providers });
```

Comment:
this is an example on all they boiler-plate we can remove by having the controller class wrap request handlers with try/catch.